### PR TITLE
Create new alert types documentation

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/3166

### CODE REVIEW

File: astro/alerts.md [hunk 1]
```diff
@@ -19,6 +19,36 @@ To configure Airflow notifications, see [Airflow email notifications](airflow-em
...
+Astro alerts can be sent to the following communication channels
+
+- Slack
+- PagerDuty
+- Email
+- DAG trigger
+
+The **DAG Trigger** communication channel works differently from other communication channel types. Instead of sending a pre-formatted alert message, Astro makes a generic request through the Airflow REST API to trigger a DAG on Astro. You can configure the triggered DAG to complete any action, such as sending a message to your own incident management system or writing data about an incident to a table.
+
+### Trigger types
```
Comment (jwitz) [inline astro/alerts.md:37]: Open to a title change here, "trigger types" is not term actually used in the product but we also don't have a UI label for this category of options

Comment (lzdanski) [reply astro/alerts.md:37]: honestly I like it - I think it's accurate and direct. Maybe someone in Product has a different idea or suggestion?

File: astro/alerts.md [hunk 2]
```diff
@@ -212,7 +236,7 @@ In the Cloud UI, you can enable alerts from the **Workspace Settings** page.
 
     - **Task duration**: Click **Task** and choose the Deployment, DAG, and task name. Enter the **Duration** for how long a task should take to run before you send an alert to your communication channels.
 
-    - **Absolute Time**: This alert triggers when a given DAG does not have a successful DAG run within a defined time window. Click **+ DAG** and choose the Deployment and the DAG that you want the alert to assess. Then, select the **Days of Week** the alert should observe, the **Verification Time** when it should look for a DAG success, and the **Lookback Period** for how long it should look back for a verification time.
+    - **Absolute Time**: Click **+ DAG** and choose the Deployment and the DAG that you want the alert to assess. Then, select the **Days of Week** the alert should observe, the **Verification Time** when it should look for a DAG success, and the **Lookback Period** for how long it should look back for a verification time.
```
Comment (lzdanski) [inline astro/alerts.md:239]:
```suggestion
    - **Absolute Time**: Click **+ DAG** and choose the Deployment and the DAG that you want the alert to assess. Then, select the **Days of Week** that the alert should observe, the **Verification Time** when it should look for a DAG success, and the **Lookback Period** for how long it should look back for a verification time.
```
I don't know why, but it feels like this is needed for parallelism with how the other things in the list are phrased.

File: astro/alerts.md [hunk 3]
```diff
@@ -19,6 +19,36 @@ To configure Airflow notifications, see [Airflow email notifications](airflow-em
 
 :::
 
+## Alert types
+
+Each Astro alert has a communication channel and a trigger type. The communication channel determines the format and destination of an alert, and the trigger type defines what causes the alert trigger. 
+
+### Communication channels
+
+Astro alerts can be sent to the following communication channels
```
Comment (lzdanski) [inline astro/alerts.md:28]:
```suggestion
You can send Astro alerts to the following communication channels
```

**Review by lzdanski — APPROVED** [review]
A couple of small suggestions!

**Pull request merged**